lib/razorpay/request.rb

```diff
@@ -80,6 +84,10 @@ def make_test_request
     def create_instance(res)
       response = res.parsed_response
 
+      if (response.is_a?(Array)==true||response.is_a?(Object)==true) && response.to_s.length == 0
```

Comment:
Use `instance_of` to check specific class. Superclass of Array is an Object. Not sure what you are checking here.

Reply:
this api returns empty json `{}` so here i am checking that if the response is empty then i will return empty json as string .

Reply:
<this was related to [this](https://razorpay.slack.com/archives/C029ZL11PNK/p1651062834979119?thread_ts=1647927334.987909&cid=C029ZL11PNK)> Pls address it